### PR TITLE
 Added info about pausing pester test on exception

### PR DESCRIPTION
**Pull request by jacekchalupka**

### CODE REVIEW

**Comment by m-kostrzewa** [comment]
As discussed IRL
* rename to Test-
* add README how to run in PerformanceTests dir, and link to it in main readme
* Accept Credentials parameter. Set defeault to (Get-Credential) in parameter list if you want

File: Test/README.md [hunk 1]
```diff
@@ -145,6 +145,31 @@ Param (
...
+
+If you need to pause execution of tests on any exception that is thrown inside 'It' block, you can make it in two ways.
+
+1. Add a breakpoint in pester function:
+
+```
+Set-PSBreakpoint -Script 'C:\Program Files\WindowsPowerShell\Modules\Pester\4.2.0\Functions\It.ps1' -Line 278
+```
+
+1. Call Suspend-PesterOnException (no parameters) at the beggining of the test script.
```
Comment (m-kostrzewa) [inline Test/README.md]:
```suggestion
1. Call Suspend-PesterOnException (no parameters) at the beggining of the test script.
```

File: Test/README.md [hunk 2]
```diff
@@ -145,6 +145,31 @@ Param (
...
+It is defined in PesterHelpers\PesterHelpers.ps1.
```
Comment (m-kostrzewa) [inline Test/README.md]:
```suggestion
It is defined in `PesterHelpers\PesterHelpers.ps1`.
```

File: Test/README.md [hunk 3]
```diff
@@ -145,6 +145,31 @@ Param (
...
+if($PSScriptRoot -match "(.*?contrail-windows-ci).*") {
+    $RepositoryRootPath = $Matches[1]
+}
+else { throw "Cannot determine repository root path" }
+
+. $RepositoryRootPath\Test\PesterHelpers\PesterHelpers.ps1
+
+Suspend-PesterOnException
+```
+
```
Comment (m-kostrzewa) [inline Test/README.md]: We also need info that suspend can be stopped by pressing Enter.

**Comment by jacekchalupka** [comment]
recheck no bug

**Pull request merged**